cyberark-secrets-provider-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
+{{- end }}
+{{- end }}
+
+{{- define "conjur.deploy-api" -}}
+{{- if .Capabilities.APIVersions.Has "batch/v1" }}
```

Comment:
If this is for `Job`, I think it is better to check specifically for it.
Like this: 
```suggestion
{{- if .Capabilities.APIVersions.Has "batch/v1/Job" }}
```

Reply:
@abrahamko are you sure it's applicable? The API version of a Job is still `batch/v1`.
Please see this [reference](https://www.k8sref.io/docs/workloads/job-v1/).

Reply:
@rafis3 Yes, I'm sure, you can see in `helm` docs here: https://helm.sh/docs/chart_template_guide/builtin_objects/
`
Capabilities.APIVersions.Has $version indicates whether a version (e.g., batch/v1) or resource (e.g., apps/v1/Deployment) is available on the cluster.
`

Reply:
Cool, thanks for sharing!